docs/usage/immutable_snapshots.md

```diff
@@ -29,3 +31,30 @@ Therefore, it is advisable to configure your garbage collection policies based o
 ## Storage Considerations
 
 Making objects immutable for extended periods can increase storage costs since these objects cannot be removed once uploaded. Storing outdated snapshots beyond their utility does not significantly enhance recovery capabilities. Therefore, consider all factors before enabling immutability for buckets, as this feature is irreversible once set by cloud providers.
+
+## Ignoring Snapshots From Restoration
+
+There might be certain cases where operators would like etcd-backup-restore to ignore particular snapshots in the object store from restoration.
```

Comment:
```suggestion
There might be certain cases where operators would like `etcd-backup-restore` to ignore particular snapshots present in the object store during restoration of etcd's data-dir.
```

Reply:
Addressed in [c3c906a](https://github.com/gardener/etcd-backup-restore/pull/781/commits/c3c906afa6b3359bd4628a7cd1afd13c0a7660e6).